package.json

```diff
@@ -19,15 +19,15 @@
     "build": "tsc && vite build",
     "watch": "vite build -w",
     "preview": "vite preview",
-    "test": "vitest",
+    "test": "npm run lint && vitest",
     "clean": "rm -rf dist node_modules yarn.lock package-lock.json",
     "reset": "npm run clean && npm install && npm run build",
     "repo-check": "REPO_VERSION=`node -e \"console.log(require('./package').version)\"` && echo TAG: v$REPO_VERSION && [[ '' == `git tag --list v${REPO_VERSION}` ]]",
     "repo-tag": "REPO_VERSION=`node -e \"console.log(require('./package').version)\"` && echo TAG: v$REPO_VERSION && git commit -a -m v$REPO_VERSION && git push && git tag v$REPO_VERSION && git push --tags;",
     "repo-publish": "npm run repo-check && npm run clean && npm install --registry https://registry.npmjs.org && npm run build && npm test && npm run repo-tag && npm publish --registry https://registry.npmjs.org",
-    "repo-publish-quick": "npm run repo-check && npm run build && npm run repo-tag && npm publish --registry https://registry.npmjs.org --access=public",
-    "lint": "npx ts-standard 'src/**/*.tsx'",
-    "lint-and-fix": "npx ts-standard --fix 'src/**/*.tsx'"
+    "repo-publish-quick": "npm run repo-check && npm run build && npm run lint && npm run repo-tag && npm publish --registry https://registry.npmjs.org --access=public",
```

Comment:
just looking at these publish scripts - let's refactor:
repo-publish should run repo-publish-quick after doing the npm install - the remaining steps are the same - see https://github.com/senecajs/seneca/blob/master/package.json#L83

repo-publish-quick needs to perform all code transformation (build, lint, etc) before testing and publishing

Reply:
I refactored `npm-publish-quick` and `npm-publish`. I also removed `lint` from `npm run test`, otherwise we'll end up linting twice on `repo-publish-quick`. But I can always add a `npm run test-and-lint` script if needed.